plugin/v1/plugin_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Google LLC
+// Copyright 2024 Google LLC
```

Comment:
Revert this change. Per go/copyright, this is the year when the file was first published.